migrations/20210328074109_highlights.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS tag (
+    name text primary key,
+    moderator integer not null,
+    content text not null,
+    official boolean not null
+);
```

Comment:
🤔 this is not the table you want to add, no?

Reply:
Bruh how did that happen